# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../siag/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur siag-3.6.0-orig/xegon/xegon.h siag-3.6.0/xegon/xegon.h
--- siag-3.6.0-orig/xegon/xegon.h	2003-01-15 23:56:54.000000000 +0000
+++ siag-3.6.0/xegon/xegon.h	2006-04-23 08:59:54.793258250 +0000
@@ -23,7 +23,7 @@
         Boolean play;
 } AppData;
 
-extern AppData app_data;
+//extern AppData app_data;
 
 /* Each list has a frame, a title, a viewport and a list */
 typedef struct egon_ui {
